### PR TITLE
Support fire_webhooks passed as a parameter to call

### PR DESCRIPTION
**Pull request by bxjx**
@DavidHuie, let me know if this is what you were thinking? Passing `fire_webhooks` parameter to `call` on a POST now sets it on the query.

### CODE REVIEW

File: spec/nationbuilder_client_spec.rb [hunk 1]
```diff
@@ -70,6 +70,21 @@
...
+        fire_webhooks: false,
+        person: {
+          email: 'bob@example.com',
+          last_name: 'Smith',
+          first_name: 'Bob'
+        }
+      }
+
+      VCR.use_cassette('parametered_post_with_fire_webhooks_false') do
+         client.call(:people, :create, params)
```
Comment (DavidHuie) [inline spec/nationbuilder_client_spec.rb]: let's create an assertion here with rspec instead of using VCR

Comment (DavidHuie) [inline spec/nationbuilder_client_spec.rb]: For example: `expect(client).to receive(...`

File: lib/nationbuilder/client.rb [hunk 2]
```diff
@@ -56,6 +56,9 @@ def raw_call(path, method, body = {}, args = {})
       request_args[:query].merge!(body)
     else
       body[:access_token] = @api_key
+      unless body[:fire_webhooks].nil?
```
Comment (DavidHuie) [inline lib/nationbuilder/client.rb]: Let's change this to `if body[:fire_webhooks]` so that it's easier to understand.

**Comment by DavidHuie** [comment]
The approach is solid, but I have a few suggestions.

**Comment by bxjx** [comment]
Sounds good. Will switch to assertions and ditch VCR

File: lib/nationbuilder/client.rb (hunk 2 above)
Comment (bxjx) [inline lib/nationbuilder/client.rb]: we'll need it to handle `fire_webhooks: false` but I can change to `if` test

Comment (DavidHuie) [inline lib/nationbuilder/client.rb]: If `body[:fire_webhooks] ` is `false`, we won't enter the block of code. `if body[:fire_webhooks]` covers all truthy and falsey values.

**Comment by bxjx** [comment]
Switched to if statement. Pretty sure it still needs that `.nil?` test or else it won't set `fire_webhooks` if `false` is passed.

Looking at it again, I think actually VCR works here unless you wanted to switch to WebMock or mock HTTPClient. I think we want to be testing that the query string is what we expect in the http call to the API.

File: lib/nationbuilder/client.rb [hunk 3]
```diff
@@ -56,6 +56,9 @@ def raw_call(path, method, body = {}, args = {})
       request_args[:query].merge!(body)
     else
       body[:access_token] = @api_key
+      if !body[:fire_webhooks].nil?
+        request_args[:query][:fire_webhooks] = body[:fire_webhooks]
```
Comment (DavidHuie) [inline lib/nationbuilder/client.rb]: Ah, I see. Let's just replace this with this nice one-liner:

```ruby
request_args[:query][:fire_webhooks] = !!body[:fire_webhooks]
```

**Comment by DavidHuie** [comment]
VCR *does* work here, but I would like the assertion to be a part of the test. Thus, let's confirm that the right stuff is passed around within the code.

File: lib/nationbuilder/client.rb (hunk 3 above)
Comment (bxjx) [inline lib/nationbuilder/client.rb]: Alas, I don't think this is quite what we want, as not setting `:fire_webhooks` in the `body` will mean that it gets set to `false`, which then means we're disabling webhooks by default.

**Comment by bxjx** [comment]
I'm not totally sure I follow on where you'd like the assertion. Please give me a sample line and I can put it in the commit. Or perhaps you can after merge?

File: lib/nationbuilder/client.rb (hunk 3 above)
Comment (DavidHuie) [inline lib/nationbuilder/client.rb]: You're totally right, man. Props for considering all of the cases.

**Comment by bxjx** [comment]
@DavidHuie how's it going? You ok to add the extra assertion or you waiting on me?

**Comment by DavidHuie** [comment]
I'll add the extra assertion. Thanks for your work, @bxjx.

**Comment by bxjx** [comment]
Thanks for maintaining and putting together the throttle code so quickly!

**Pull request merged**